tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3906,12 +3919,16 @@ <h2>Show similarity to selected datapoint</h2>
           } else {
             this.comparedIndices = [];
             this.counterfactualExampleAndInference = null;
-            const temp = this.selectedExampleAndInference;
-            this.selectedExampleAndInference = null;
-            this.selectedExampleAndInference = temp;
+            this.refreshSelectedDatapoint_();
           }
         },
 
+        refreshSelectedDatapoint_: function() {
+          const temp = this.selectedExampleAndInference;
+          this.selectedExampleAndInference = null;
+          this.selectedExampleAndInference = temp;
```

Comment:
This may cause two cycles of renders. Instead, would `this.set('selectedExampleAndInference', this.selectedExampleAndInference)` work? Polymer does not have any shallow equals check to shortcircuit a render.

Reply:
just checked. this.set('selectedExampleAndInference', this.selectedExampleAndInference)  does not cause the actual visual update